src/main/resources/com/google/api/codegen/ruby/smoke_test.snip

```diff
@@ -23,7 +23,15 @@ describe "{@smokeTest.name}" do
       project_id = ENV["SMOKE_TEST_PROJECT"].freeze
     @end
 
-    {@sampleCodeCore(smokeTest.apiMethod)}
+    process_response = ->(response) {
```

Comment:
I can see the reason why you want to directly re-use samplegen code as a drop-in, but I suspect that may not work, for example, in LRO, the sample code has 'operation.wait_until_done!' which is a block call, in the unit-test case, you might not want that. After all, the unit test here is just sort of compiler check for the API calls. There is no need to test operation.reload! or operation.wait_until_done! So I think you may end up with tweaking a lot of samplegen code. Instead, I would focus on the code to make API calls.